### DISCUSSION TITLE
Placeholder text?

### DISCUSSION THREAD

**Discussion by FezVrasta** [excalidraw/excalidraw#3987] [General]
Hi, I can't find any way to insert dummy/placeholder text into my drawings. I'd like to just display some scribbles or solid blocks that give the idea of text, do you think it's something you could add to the software?

Example:

![image](https://user-images.githubusercontent.com/5382443/133994065-1b4a4c7b-f069-4f32-b061-d046a14f0075.png)

**Comment by dwelle** [comment]
I think this makes sense. It could be the 4th `font family`, if the idea that you'd type anything and the letters would be scribbles.

I think there was a feature request in https://github.com/excalidraw/excalidraw/issues for this but can't find it right now

**Comment by ad1992** [comment]
We have a similar request in https://github.com/excalidraw/excalidraw/issues/3635

**Reply by dwelle** [reply]
that's the one I was thinking about, thanks
